ocfweb/api/lab.py

```diff
@@ -0,0 +1,41 @@
+from django.http import JsonResponse
+from ocflib.infra.hosts import hostname_from_domain
+from ocflib.lab.stats import get_connection
+from ocflib.lab.stats import list_desktops
+
+from ocfweb.caching import cache
+from ocfweb.caching import periodic
+
+
+@cache()
+def _enumerate_desktops():
+    return list_desktops(public_only=True)
+
+
+@periodic(15)
+def _get_active_desktops():
```

Comment:
what is an active desktop? not clear from the name if this means any available desktop or a desktop currently in use.

can we just call it `get_desktops_currently_used` or something?